src/index.html

```diff
@@ -1,13 +1,301 @@
+                      alt="Range & Integrated lights"
+                      class="category__photo photo"
+                    >
+                  </a>
+                </div>
+
+                <h3 class="category__title">Range & Integrated lights</h3>
+                <div class="grid__wrapper">
+                  <p class="category__description">
+                    The removable battery has up to 70km battery autonomy and weighs only 2.4 kg.
+                    Lights integrated into the frame give you always-on visibility day and night.
+                  </p>
+                </div>
+              </article>
+
+              <article class="category">
+                <div class="category__photos">
+                  <a href="#" class="category__link category__link--square">
+                    <img
+                      src="./images/Hydraulic disc brakes & Lightweight-1.png"
+                      alt="Hydraulic disc brakes & Lightweight"
+                      class="category__photo photo"
+                    >
+                  </a>
+
+                  <a href="#" class="category__link category__link--wide">
+                    <img
+                      src="./images/Hydraulic disc brakes & Lightweight-2.png"
+                      alt="Hydraulic disc brakes & Lightweight"
+                      class="category__photo photo"
+                    >
+                  </a>
+                </div>
+
+                <h3 class="category__title">Hydraulic disc brakes & Lightweight</h3>
+                <div class="grid__wrapper">
+                  <p class="category__description">
+                    Brakes with total stopping power the second you make contact.
+                    The removable battery has up to 70km battery autonomy and weighs only 2.4 kg.
+                    Lights integrated into the frame give you always-on visibility day and night.
+                  </p>
+                </div>
+              </article>
+
+              <div class="grid__wrapper">
+                <a href="#" class="categories__button">Explore</a>
+              </div>
+            </div>
+          </section>
+
+          <section class="contacts" id="contacts">
+
+            <h2 class="section-title">Contact us</h2>
+
+            <div class="contacts__wrapper">
+
+              <form class="contacts__inputs" onsubmit="this.reset(); return false;">
+                <input
+                    type="text"
+                    name="name"
+                    placeholder="Name"
+                    class="contacts__input"
+                    required
+                >
+
+                <input
+                    type="email"
+                    name="email"
+                    placeholder="Email"
+                    class="contacts__input"
+                    required
+                >
+
+                <textarea
+                  name="message"
+                  type="text"
```

Comment:
Attribute type not allowed on element [textarea](https://html.spec.whatwg.org/multipage/#the-textarea-element) at this point.